sdk/include/opentelemetry/sdk/trace/fork_aware_span_processor.h

```diff
@@ -0,0 +1,88 @@
+// The MIT License (MIT)
+
+// Copyright (c) 2015-2016 LightStep
```

Comment:
@pyohannes @reyang 
I have a licensing question here. The rest of OpenTelemetry code is Apache.. I believe we had a concern once adding dependency on other MIT-licensed libraries, such as MS Guidelines Support Library (which is also MIT btw).... Are we OK now to change the policy and allow adding other company -licensed contributions, with their respected copyrights that do not match the license here: https://github.com/open-telemetry/opentelemetry-cpp/blob/master/LICENSE